packages/apollo-engine-reporting/src/contextualizedStats.ts

```diff
@@ -0,0 +1,231 @@
+import { DurationHistogram } from './durationHistogram';
+import {
+  IStatsContext,
+  Trace,
+  TypeStat,
+  QueryLatencyStats,
+  IPathErrorStats,
+} from 'apollo-engine-reporting-protobuf';
+
+export class ContextualizedStats {
+  statsContext: IStatsContext;
+  queryLatencyStats: QueryLatencyStats;
```

Comment:
Can we do make this be a IQueryLatencyStats, or better still a class we define in this file that implements IQueryLatencyStats but has fields typed as DurationHistogram , so that we don't need a bunch of casts later?

Reply:
Made a `IQueryLatencyStats` interface and removed the Duration Histogram casting.